src/coreclr/jit/llvmcodegen.cpp

```diff
@@ -621,33 +527,36 @@ void Llvm::generateEHDispatch()
         }
 
         if (((model == CorInfoLlvmEHModel::Cpp) || (model == CorInfoLlvmEHModel::Emulated)) &&
-            (outerDispatchLlvmBlock != nullptr))
+            (outerUnwindLlvmBlock != nullptr))
         {
-            // We have the "dispatch pad" block. Since we're generating dispatches from outer to inner, we already have
-            // the "actual" dispatch block; it will be the next one.
-            outerDispatchLlvmBlock = outerDispatchLlvmBlock->getNextNode();
-            assert(outerDispatchLlvmBlock != nullptr);
+            // We have the "unwind" block. Since we're generating things from outer to inner, we already have its
+            // "local" counterpart; it will be the next block.
+            outerUnwindLlvmBlock = outerUnwindLlvmBlock->getNextNode();
+            assert(outerUnwindLlvmBlock != nullptr);
```

Comment:
I have switched the terminology away from "dispatch" to "unwind" in a bunch of places because it better represents that this code is involved in handling the second pass ("unwinding the stack") as opposed to the first pass ("dispatching the exception") as it was originally.